client_side/.infrastructure/prompts/ending.txt

```diff
@@ -0,0 +1,2 @@
+Congratulations! You have completed the interactive portion of the, experiment,
```

Comment:
I think it would be best to integrate these into the main scripts (even if that causes lines longer than 80 characters long).